library/aos_version.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/python
```

Comment:
I don't have an opinion here, but a bit confused.

A previous suggestion was exactly on the other direction, to move this to the top-level so that it could be "potentially used" by other roles.

Looking at the repository today one can see cases of both: symlinks and modules "hidden" inside of roles. What is the practice we're generally trying to follow? IOW, what should I be thinking about when making this decision myself next time?